Configuration/TCA/tx_books_domain_model_books.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+return [
+    'ctrl' => [
+        'label' => 'title',
+        'title' => 'LLL:EXT:books/Resources/Private/Language/Tca.xlf:books',
+        'delete' => 'deleted',
+        'crdate' => 'crdate',
+        'tstamp' => 'tstamp',
+        'hideAtCopy' => true,
+        'prependAtCopy' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.prependAtCopy',
+        'cruser_id' => 'cruser_id',
+        'useColumnsForDefaultValues' => 'hidden',
+        'enablecolumns' => [
+            'disabled' => 'hidden',
+        ],
+        'iconfile' => 'EXT:books/Resources/Public/Icons/Book.svg',
+    ],
+    'columns' => [
+        'title' => [
+            'l10n_mode' => 'prefixLangTitle',
+            'label' => 'LLL:EXT:books/Resources/Private/Language/Tca.xlf:books.title',
+            'config' => [
+                'type' => 'input',
+                'size' => 50,
+                'max' => 255,
+                'eval' => 'trim,required'
+            ]
+        ],
+        'year_of_publication' => [
+            'label' => 'LLL:EXT:books/Resources/Private/Language/Tca.xlf:books.year_of_publication',
+            'config' => [
+                'type' => 'input',
+                'size' => 4,
+                'max' => 4,
+                'eval' => 'trim,int,required'
+            ]
+        ]
```

Comment:
Please let's always have a comma after the last array element for multi-line arrays. This will keep the Git diffs smaller when lines are added or removed.